include/sst/filters/CytomicSVF.h

```diff
@@ -81,26 +81,27 @@ struct CytomicSVF
     SIMD_M128 negoneSSE{SETALL(-1.0)};
     SIMD_M128 twoSSE{SETALL(2.0)};
     SIMD_M128 negtwoSSE{SETALL(-2.0)};
+
     enum Mode
     {
-        LP,
-        HP,
-        BP,
-        NOTCH,
-        PEAK,
-        ALL,
-        BELL,
-        LOW_SHELF,
-        HIGH_SHELF
+        Lowpass,
+        Highpass,
```

Comment:
So if we are going to break all the clients let’s also make this an enum strict so we require the mode prefix and don’t have ambiguity with the free symbol Lowpass etc